lua/diffview/vcs/init.lua

```diff
@@ -1,3 +1,7 @@
-local git = require('diffview.vcs.adapters.git.utils')
+local M = {}
 
-return git
+function M.get_adapter(path)
```

Comment:
Idea is that the lib calls this `get_adapter` with the current path, this is used to determine which kind of repo we are in.

For now I will make this return the `GitAdapter` so the existing features can be restored first.